### DISCUSSION TITLE
Normalised Gene Activity matrix still affected by peak region fragments

### DISCUSSION THREAD

**Discussion by wudustan** [stuart-lab/signac#548] [Q&A] [answered]
Following the vignette I generate my Gene Activity matrix from my scATAC dataset. I then NormalizeData() using the standard approach. 

However, when I plot genes, I can clearly see that parts of the library with higher **peak_region_fragments** values also have higher gene activity scores after normalisation. 

Is it possible to correct this? 

Example:

**Peak region fragments plot**
<img width="542" alt="Screenshot 2021-04-01 at 15 05 55" src="https://user-images.githubusercontent.com/15104192/113306404-f5031a80-92fb-11eb-99bc-1f5e645a8f00.png">

**Gene signature from normalized Gene Activity** 
<img width="567" alt="Screenshot 2021-04-01 at 15 08 15" src="https://user-images.githubusercontent.com/15104192/113306540-1cf27e00-92fc-11eb-8f1f-27fdcb99fcf3.png">

**Comment by timoast** [comment, marked as answer]
It's difficult to fully remove these differences due to different sequencing depth using a simple method like that used by `NormalizeData()`. However, for most downstream applications, these differences are not usually an issue since the biological variation is typically a much stronger signal and are consistent across datasets.

**Reply by wudustan** [reply]
Do you have a recommendation for a package / method to use the peak_region_fragments as a latent variable to compare the gene expression .

**Reply by timoast** [reply]
Yes, Seurat. You can include a latent variable in a few of the DE methods in `FindMarkers()`, eg `FindMarkers(..., test.use="LR", latent.vars="peak_region_fragments")`
